src/constraints/util/typedArray.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	isBigInt64Array,
+	isBigUint64Array,
+	isFloat32Array,
+	isFloat64Array,
+	isInt16Array,
+	isInt32Array,
+	isInt8Array,
+	isTypedArray,
+	isUint16Array,
+	isUint32Array,
+	isUint8Array,
+	isUint8ClampedArray
+} from 'node:util/types';
```

Comment:
I think this will remove browser compat, thoughts @favna @kyranet ?

Reply:
I can change all these checks to `instanceOf` check but for `TypedArray` should I check `instanceOf` with all the types?

Reply:
seems like https://stackoverflow.com/questions/15251879/how-to-check-if-a-variable-is-a-typed-array-in-javascript#comment73008682_29651223 one works though.